entrypoint.sh

```diff
@@ -134,14 +134,21 @@ echo "${DOMAIN}" > /etc/letsencrypt/san_list
 mkdir -p /etc/letsencrypt/webrootauth/
 
 # Template a cronjob to renew certificate with the webroot authenticator
-CRONFILE=/etc/cron.daily/renew
-cat <<EOF >${CRONFILE}
+RENEW=/renew
+cat <<EOF >${RENEW}
 #!/bin/sh
 # First renew certificate, then reload nginx config
 certbot renew --webroot --webroot-path /etc/letsencrypt/webrootauth/ --post-hook "/usr/sbin/nginx -s reload"
 EOF
+chmod +x ${RENEW}
 
-chmod +x ${CRONFILE}
+# Install crontab
+echo "0 0 * * * /renew" | crontab
+echo "Created crontab:"
```

Comment:
When reading the log, this entry may not be really helpful. Can you make it more precise ?

Reply:
Mmm, something like "Created crontab to renew certificates"?

Not sure that's that much more helpful given that the command is `/renew` :D

Or what did you have in mind?

Reply:
if the name of the command shows up in the log at execution then I guess it's fine :)

Reply:
I should re-use `${CRONFILE}` to set it though, I've just realised. Let me fix that.